presentation/src/main/java/com/keeply/presentation/core/components/TabBar.kt

```diff
@@ -0,0 +1,120 @@
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Tab
+import androidx.compose.material3.TabRow
+import androidx.compose.material3.TabRowDefaults.tabIndicatorOffset
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.keeply.presentation.core.theme.KeeplyTheme
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+
+@Composable
+fun KeeplyTab(
+    selectedTabIndex: Int,
+    tabs: ImmutableList<String>,
+    modifier: Modifier = Modifier,
+    onClick: (Int) -> Unit,
+) {
+    Box {
+        HorizontalDivider(
+            modifier = modifier
+                .fillMaxWidth()
+                .height(2.dp)
+                .align(Alignment.BottomCenter)
+        )
+
+        TabRow(
+            modifier = Modifier
+                .padding(horizontal = 16.dp)
+                .fillMaxWidth(),
+            selectedTabIndex = selectedTabIndex,
+            containerColor = Color.Transparent,
+            indicator = { tabPositions ->
+                if (tabPositions.isNotEmpty()) {
+                    Box(
+                        Modifier
+                            .tabIndicatorOffset(tabPositions[selectedTabIndex])
+                            .fillMaxWidth()
+                            .height(2.dp)
+                            .background(
+                                color = KeeplyTheme.colors.neutralBlack
+                            )
+                    )
+                }
+            }
+        ) {
+            tabs.forEachIndexed { index, title ->
+                val isSelected = selectedTabIndex == index
+
+                Tabs(
+                    title = title,
+                    index = index,
+                    isSelected = isSelected,
+                    onClick = onClick
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun Tabs(
+    title: String,
+    index: Int,
+    isSelected: Boolean,
+    onClick: (Int) -> Unit
+) {
+    Tab(
+        selected = isSelected,
+        onClick = { onClick(index) },
+        modifier = Modifier
+            .height(48.dp),
+        content = {
+            val titleColor: Color = if (isSelected) {
+                KeeplyTheme.colors.neutralBlack
+            } else {
+                KeeplyTheme.colors.neutral400
+            }
+
+            KeeplyText(
+                modifier = Modifier
+                    .padding(
+                        top = 12.dp
+                    ),
```

Comment:
p3: padding을 keeplyText대신 Tab에 주는게 더 디자인 가이드에 가까울 것 같습니다!
```suggestion
    Tab(
        selected = isSelected,
        onClick = { onClick(index) },
        modifier = Modifier
            .height(48.dp)
            .padding(
                top = 12.dp
            ),
        content = {
            val titleColor: Color = if (isSelected) {
                KeeplyTheme.colors.neutralBlack
            } else {
                KeeplyTheme.colors.neutral400
            }

            KeeplyText(
                modifier = Modifier,
```

<img width="400" alt="image" src="https://github.com/user-attachments/assets/1e6502de-524a-4838-b814-225f21bab825" />
<img width="400" alt="image" src="https://github.com/user-attachments/assets/0a052404-2508-450b-b5fa-db761637d0c7" />

Reply:
@euzl 해당 사항 및 develop pull confilct 이슈 반영해서 수정했어요

Reply:
감사합니다